test/tests/functional/pbs_reservations.py

```diff
@@ -173,3 +173,30 @@ def test_not_honoring_resvs(self):
 
         self.server.expect(JOB, {'job_state': 'Q'}, id=j1id)
         self.server.expect(JOB, {'job_state': 'Q'}, id=j2id)
+
+    def test_sched_cycle_starts_on_resv_end(self):
+        """
+        This test checks whether the sched cycle gets started
+        when the advance reservation ends.
+        """
+        now = int(time.time())
+        a = {'Resource_List.select': "1:ncpus=2",
+             'reserve_start': now + 5,
+             'reserve_end': now + 12,
+             }
+        r = Reservation(TEST_USER, a)
+        rid = self.server.submit(r)
+        attr = {'Resource_List.walltime': '00:00:20'}
+        j = Job(TEST_USER, attr)
+        jid = self.server.submit(j)
+        self.server.expect(JOB, {ATTR_state: 'Q'},
+                           id=jid)
```

Comment:
Could you please merge the above two lines.  They should be under the 79 char limit that pep8 requires.